core/storage/classifier/gae_models_test.py

```diff
@@ -387,9 +379,9 @@ def test_get_model_from_exploration_attributes(self):
         self.assertDictEqual(
             mappings[0].algorithm_ids_to_job_ids, {'algorithm_id': job_id})
 
-    def test_create_multi_mappings(self):
-        # type: () -> None
-        state_training_jobs_mappings = [] # type: List[classifier_domain.StateTrainingJobsMapping]
+    def test_create_multi_mappings(self) -> None:
+        state_training_jobs_mappings: (
+            List[classifier_domain.StateTrainingJobsMapping]) = []
```

Comment:
Are the `(`, `)` needed here?

Reply:
I think they would be needed because the type is too long to fit in the above line, and for this particular case the type cannot be broken down into multiple lines.

Reply:
Yes I wanted to do this only, forgot to change this place. Changed now.